.github/workflows/ci.yml

```diff
@@ -74,10 +74,11 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        version: ["v1_20","v1_19","v1_18","v1_17","v1_16"]
+        version: ["v1_20"]
         namespace: ["default","flink"]
-        mode: ["native", "standalone"]
-        java-version: [ 11, 17, 21 ]
```

Comment:
I think this will also get rid of our existing tests. Maybe what you need is an exclusion like in https://github.com/apache/flink-kubernetes-operator/blob/main/.github/workflows/ci.yml#L104

Reply:
Yeah. I've been trying to run the workflow on my fork to validate it rather than waiting for approval to run on the main flink operator repo. But to try and keep things comprehensible I've tried to turn most of the suite off while I work out if it's building and running properly.

My hope is to drop the final two commits before this actually gets merged.

Reply:
To be a bit more explicit [ca99f0f](https://github.com/apache/flink-kubernetes-operator/pull/881/commits/ca99f0fde68f5dd4a95a890396d9ee2d368e6cff) and [aa96389](https://github.com/apache/flink-kubernetes-operator/pull/881/commits/aa963890c6524d7c3ec1579dd943d536d16e0d72) will be dropped before this is mergeable, but I'm hoping the rest of the PR stays the same. 

Trying to find my 4 jobs in the other 2-4 hundred was a bit to painful

Reply:
makes sense!